src/components/Kv/KvHero.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+	<div class="hero">
+		<div class="images">
+			<div class="images-placeholder"></div>
+			<div class="images-container">
+				<slot name="images"></slot>
+			</div>
+		</div>
+		<div class="headline">
```

Comment:
For the 3rd phase I we'll prob want another computed check against "all" of the headline slots here so we turn on and off the block.